.gitignore

```diff
@@ -19,6 +19,8 @@ py27_gevent/
 py27_native/
 py3[0-9]_gevent/
 py3[0-9]_native/
+a.out
+src/python/grpcio_*/LICENSE
```

Comment:
Does `src/python/grpcio` also create a license?

Reply:
Nope. It uses the root directory one.